__block-templates/query-grid.html

```diff
@@ -0,0 +1,35 @@
+<!-- wp:query { "query":{ "inherit": true }, "displayLayout": { "type": "flex", "columns":4 } } -->
```

Comment:
I preserved this grid template as it was already in the POC. For now the code will use only the list view.